### PR TITLE
DRILL-5943: Avoid the strong check introduced by DRILL-5582 for PLAIN…

### PR DESCRIPTION
**Pull request by sohami**
… mechanism

### CODE REVIEW

**Comment by sohami** [comment]
@parthchandra & @laurentgo - Please help to review this PR.

File: contrib/native/client/src/clientlib/saslAuthenticatorImpl.hpp [hunk 1]
```diff
@@ -59,6 +59,12 @@ class SaslAuthenticatorImpl {
 
     const char *getErrorMessage(int errorCode);
 
+    static const std::string KERBEROS_SIMPLE_NAME;
+
+    static const std::string KERBEROS_SASL_NAME;
```
Comment (laurentgo) [inline contrib/native/client/src/clientlib/saslAuthenticatorImpl.hpp]: do we need to expose it? (it looks like we only look for the keys)

**Comment by sohami** [comment]
@laurentgo  - Thanks for the review. Updated PR based on comment.

**Comment by priteshm** [comment]
@parthchandra can you please review this?

**Pull request closed without merging**